.prettierignore

```diff
@@ -0,0 +1,117 @@
+# Ignored files identified within .gitignore:
+.cache/clangd
+/vendor/*
+.DS_Store
+.brave_gclient_*
+.gclient_*
+.tags*
+.cipd
+/.idea/
+/components/brave_new_tab_ui/data/LICENSE
+/components/brave_webtorrent/extension/out/
+/components/brave_rewards/resources/extension/brave_rewards/out/
+/components/third_party/adblock/LICENSE
+/components/third_party/local_data/LICENSE
+/dist/
+/out/
+node_modules/
+patches/**/*.patchinfo
```

Comment:
this seems to include explicit files that would never be touched by prettier. I think it can be greatly simplified - is it just a copy of .gitignore?

Reply:
I did use .gitignore as a starting point

Reply:
Simplified. Please let me know if there are other entries you would like removed